test/test_scp.rb

```diff
@@ -328,7 +328,7 @@ def test_pipe
     IO.pipe do |read_io, write_io|
       if fork
         write_io.close
-        Oj.sc_parse(handler, read_io) {|v| p v}
+        Oj.sc_parse(handler, read_io)
```

Comment:
@ohler55 this one was outputing to tests, I looks like this was unwanted. Not 100% sure though